search/domain/service.go

```diff
@@ -36,13 +36,20 @@ type (
 		SortOptions    []SortOption
 	}
 
-	// SortOption defines how sorting is possible, with both an asc and desc option
+	// SortOption defines how sorting is possible, and which of them are activated with both an asc and desc option
 	SortOption struct {
-		Label        string
-		Asc          string
-		Desc         string
-		SelectedAsc  bool
+		// Label that you normally want to show in the frontend (e.g. "Price")
+		Label string
+		// Field that you need to use in SearchRequest>SortFilter
+		Field string
+		// SelectedAsc true if sorting by this field is active
+		SelectedAsc bool
+		// SelectedDesc true if sorting by this field is active
 		SelectedDesc bool
+		// Asc - DEPRECATED - used to give the "Field" that is used to trigger ascending search
```

Comment:
I updated the deprecated fields. Which fields do you mean? Field or Asc/Desc? We left Asc/Desc there because we weren't sure if we introduce a breaking change if we remove them completely. In case of Field. We are using it for the graphql part and it is used in flamingo-om3 which is in the other merge request.

Reply:
@danielpoe could both fields ( `Asc`, `Desc`) actually be deleted? this looks quite like dead code.

Reply:
How do you know they are not used??
They are part of the public API - projects depend on it and its also filled in the Adapters currently
@panzerfahrer

Reply:
The team decided to deprecate the field. If the fields are actually in use, we should probably add fallback behavior. Or do we need a rollback with another solution?

Reply:
... @kalypso-byte could you please clarify, whether the deprecation was a valid move.

Reply:
The DEPRECATED is valid  we did this together. - but because the intent of this property is wrong (it represents the fieldname for searchperience)
We just need time to adjust the current usage

Reply:
So all is fine at the moment - just wanted to clarify